Clockwise/CircuitBreakerStateDescriptor.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+
+namespace Clockwise
+{
+    public class CircuitBreakerStateDescriptor
+    {
+        protected bool Equals(CircuitBreakerStateDescriptor other)
```

Comment:
Do we really need the equality overrides?

Reply:
to compare states